spring-cloud-netflix-core/pom.xml

```diff
@@ -199,4 +199,25 @@
 			<optional>true</optional>
 		</dependency>
 	</dependencies>
+	<profiles>
+		<profile>
+			<id>java8plus</id>
+				<activation>
+					<jdk>[1.8,2.0)</jdk>
```

Comment:
OK, we require a java 8 build anyway (one optional module requires it), so this will always be true.